flake.nix

```diff
@@ -187,6 +187,7 @@
           # Define lambda that returns a devShell derivation with extra test-required packages
           # given the bpftrace package derivation as input
           mkBpftraceDevShell =
+            llvmVersion:
```

Comment:
Try to drop `pkg` below as well? Can do fmt string selection using `llvmVersion`. Would be good to have non-duplicated params.